TestApp/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
```

Comment:
same here, do not manually edit this file.

Reply:
Let's remove the timestamp to avoid confusing because sometime you have to manually edit this file and timestamp get out of sync quickly.

Reply:
It will be reinserted next time there is an update but that's ok for now I guess, it does not matter.